packages/focuslock/src/react/__specs__/index.spec.js

```diff
@@ -3,15 +3,113 @@ import { mount } from 'enzyme'
 
 import FocusLock from '../index'
 
-describe('FocusLock', () => {
-  let props
+const children = (
+  <ul>
+    <li>
+      <a href="#">first</a>
+    </li>
+    <li>
+      <a href="#">second</a>
+    </li>
+    <li>
+      <a href="#">third</a>
+    </li>
+    <li>
+      <a href="#">forth</a>
+    </li>
+  </ul>
+)
 
+describe('FocusLock', () => {
   beforeEach(() => {
-    props = {
-      children: 'children'
-    }
+    document.activeElement.blur()
+  })
+
+  it('has defaultProps', () => {
+    expect(FocusLock.defaultProps).toMatchInlineSnapshot(`
+Object {
+  "as": "div",
+  "autofocus": true,
+  "returnFocus": true,
+}
+`)
+  })
+
+  describe('WITH focusable children', () => {
+    it('renders successfully', () => {
+      expect(() => mount(<FocusLock>{children}</FocusLock>)).not.toThrow()
+    })
+  })
+
+  describe('WITHOUT focusable children', () => {
```

Comment:
I love these test cases.  In other components, I may have put these in snapshot test, generated from storybook.  But on thinking about what you're asserting here, that different children scenarios are *possible* without some child state to check against probably fits well into this more traditional unit test.